src/Components/Patient/PatientRegister.tsx

```diff
@@ -800,7 +800,10 @@ export const PatientRegister = (props: PatientRegisterProps) => {
             : null
           : null,
       test_type: formData.test_type,
-      name: formData.name,
+      name: formData.name
+        .split(" ")
+        .map((i: string) => i.charAt(0).toUpperCase() + i.slice(1))
+        .join(" "),
```

Comment:
We could instead make use of html's autocapitalize right? That way it'll also be visible to the user of what exactly is being sent to the backend too.

Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize

Reply:
> autocapitalize doesn't affect behavior when typing on a physical keyboard. It affects the behavior of other input mechanisms such as virtual keyboards on mobile devices and voice input.

Autocapitalize doesn't work on desktop I think. The MDN examples weren't working for me. Also no safari support : (

Reply:
We could use the capitalize method from loadash instead...